pkg/tuple/tuple.go

```diff
@@ -325,3 +325,64 @@ func IsTypedWildcard(s string) bool {
 func TypedPublicWildcard(objectType string) string {
 	return BuildObject(objectType, Wildcard)
 }
+
+// MustParseTupleString attempts to parse a relationship tuple specified
+// in string notation and return the protobuf TupleKey for it. If parsing
+// of the string fails this  function will panic. It is meant for testing
+// purposes.
+//
+// Given string 'document:1#viewer@user:jon', return the protobuf TupleKey
+// for it.
+func MustParseTupleString(s string) *openfgav1.TupleKey {
+	t, err := ParseTupleString(s)
+	if err != nil {
+		panic(err)
+	}
+
+	return t
+}
+
+func MustParseTupleStrings(tupleStrs ...string) []*openfgav1.TupleKey {
+	tuples := make([]*openfgav1.TupleKey, 0, len(tupleStrs))
+	for _, tupleStr := range tupleStrs {
+		tuples = append(tuples, MustParseTupleString(tupleStr))
+	}
+
+	return tuples
+}
+
+// ParseTupleString attempts to parse a relationship tuple specified
+// in string notation and return the protobuf TupleKey for it. If parsing
+// of the string fails this  function returns an err.
+//
+// Given string 'document:1#viewer@user:jon', return the protobuf TupleKey
+// for it or an error.
+func ParseTupleString(s string) (*openfgav1.TupleKey, error) {
```

Comment:
Thanks for adding this 🙇‍♀️  I preferred if you submitted to `main` though, so as not to make ListUsers PR even bigger 😅

Reply:
My thoughts there - the tests utilizing these changes are only applicable to the scope of this PR right now, but I'm happy to move it to main ahead of the curve if that's the preference. Sounds like that is, so I'll just bite the bullet and do so.

Reply:
https://github.com/openfga/openfga/pull/1586 - moved over to a PR targeting `main`.